src/shared/particle_dynamics/particle_dynamics_algorithms.h

```diff
@@ -198,16 +199,15 @@ class InteractionSplit : public BaseInteractionDynamics<LocalDynamicsType, Paral
 {
   protected:
     RealBody &real_body_;
-    SplitCellLists &split_cell_lists_;
+    CellLinkedList &cell_linked_list_;
```

Comment:
I can't find a way to make this function applicable to `BaseCellListList`, might need to write a new `SplitInteraction` function for `MultiLevelCellLinkedList`